src/connector/src/source/manager.rs

```diff
@@ -25,14 +27,36 @@ pub struct SourceColumnDesc {
     pub data_type: DataType,
     pub column_id: ColumnId,
     pub fields: Vec<ColumnDesc>,
-    /// Now `skip_parse` is used to indicate whether the column is a row id column.
-    pub is_row_id: bool,
+    pub column_type: SourceColumnType,
 
-    pub is_meta: bool,
     // `is_pk` is used to indicate whether the column is part of the primary key columns.
     pub is_pk: bool,
 }
 
+#[derive(Clone, Debug, PartialEq)]
+pub enum SourceColumnType {
+    Normal,
+
+    // internal columns
```

Comment:
Would you please add some docs for these variants of internal columns and show when they'll be used?

Reply:
added